### PR TITLE
new zarr paths

### PR DESCRIPTION
**Pull request by hardingnj**

### CODE REVIEW

**Comment by hardingnj** [comment]
removed `zarr2` reference in favour of `zarr/all` etc

**Comment by hardingnj** [comment]
Can I get a review on this @malariagen/vector-core ?

It should be uncontroversial, but it will break things if people are using the now deprecated `zarr2` directory in phase2.

File: src/python/ag1k/phase2_ar1.py [hunk 1]
```diff
@@ -143,7 +143,7 @@ def init(release_dir, load_geneset=False, geneset_attributes=None):
     # main callset, PASS variants only
     callset_pass_h5_fn = os.path.join(variation_dir, 'main', 'hdf5', 'pass', 'ag1000g.phase2.ar1.pass.h5')
     callset_pass_lite_h5_fn = os.path.join(variation_dir, 'main', 'hdf5', 'lite', 'ag1000g.phase2.ar1.pass.lite.h5')
-    callset_pass_zarr_fn = os.path.join(variation_dir, 'main', 'zarr2', 'ag1000g.phase2.ar1.pass')
+    callset_pass_zarr_fn = os.path.join(variation_dir, 'main', 'zarr', 'pass', 'ag1000g.phase2.ar1')
```
Comment (alimanfoo) [inline src/python/ag1k/phase2_ar1.py]:
```suggestion
    callset_pass_zarr_fn = os.path.join(variation_dir, 'main', 'zarr', 'pass', 'ag1000g.phase2.ar1.pass')
```

Still need to include ".pass" in the final path segment.

File: src/python/ag1k/phase2_ar1.py [hunk 2]
```diff
@@ -156,7 +156,7 @@ def init(release_dir, load_geneset=False, geneset_attributes=None):
     # main callset, PASS biallelic variants only
     callset_pass_biallelic_h5_fn = os.path.join(variation_dir, 'main', 'hdf5', 'biallelic', 'ag1000g.phase2.ar1.pass.biallelic.h5')
     callset_pass_biallelic_lite_h5_fn = os.path.join(variation_dir, 'main', 'hdf5', 'lite', 'ag1000g.phase2.ar1.pass.biallelic.lite.h5')
-    callset_pass_biallelic_zarr_fn = os.path.join(variation_dir, 'main', 'zarr2', 'ag1000g.phase2.ar1.pass.biallelic')
+    callset_pass_biallelic_zarr_fn = os.path.join(variation_dir, 'main', 'zarr', 'biallelic', 'ag1000g.phase2.ar1')
```
Comment (alimanfoo) [inline src/python/ag1k/phase2_ar1.py]:
```suggestion
    callset_pass_biallelic_zarr_fn = os.path.join(variation_dir, 'main', 'zarr', 'biallelic', 'ag1000g.phase2.ar1.pass.biallelic')
```

**Review by alimanfoo — COMMENTED** [review]
Thanks Nick, couple of fixes suggested.

Btw there are also [new data catalogs](https://github.com/malariagen/vector-tools/blob/master/catalogs/ag1000g/local.py) in the vector-tools repo, but still worth merging this PR while agam-report-base is still in use.

**Comment by hardingnj** [comment]
Good catches- thanks!

**Comment by alimanfoo** [comment]
Thanks Nick.

**Pull request merged**